cfpb/capital-framework-sandbox
# Hero updates

The Heros have been updated to match the latest spec from the Design Manual. This update required slight markup changes and bumping to the latest version of Capital Framework.

## Changes

- Updated markup to match CF examples

## Testing

- Start the server with `npm start`
- Navigate to `http://localhost:3000/components/cf-layout/heroes.html` and check the Hero examples.

## Review

- @Scotchester 
- @sebworks 
- @anselmbradford 

[Preview this PR without the whitespace changes](?w=0)

## Screenshots

![localhost-3000-components-cf-layout-heroes html](https://cloud.githubusercontent.com/assets/1280430/25442497/cd879f88-2a6a-11e7-992a-65a4d8f57a8b.png)

## Checklist
